### PR TITLE
Make metadata reports available to local users

### PR DESCRIPTION
**Pull request by hanars**
Similar to what we did for the individual metadata report, moves the family and varant metadata reports from the reports tab to the summary data tab and makes them available to anyone. Mostly involves copy-pasting code

### CODE REVIEW

File: seqr/views/apis/summary_data_api.py [hunk 1]
```diff
@@ -348,3 +349,122 @@ def _get_airtable_collaborator_names(user, collaborator_ids):
         collaborator_id: collaborator_map.get(collaborator_id, {}).get('CollaboratorID')
         for collaborator_id in collaborator_ids
     }
+
+
+@login_and_policies_required
+def family_metadata(request, project_guid):
+    projects, _ = _get_metadata_projects(request, project_guid)
```
Comment (hanars) [inline seqr/views/apis/summary_data_api.py:356]: copy-pasted from report_api, this is the only line that changed (uses a different helper function to get the allowed projects)

File: seqr/views/apis/summary_data_api.py [hunk 2]
```diff
@@ -348,3 +349,122 @@ def _get_airtable_collaborator_names(user, collaborator_ids):
...
+def _get_family_structure(num_individuals, num_known_individuals):
+    if (num_individuals and num_known_individuals == num_individuals) or (
+            num_known_individuals in {0, 3} and num_individuals == num_known_individuals + 1):
+        return FAMILY_STRUCTURES[num_individuals]
+    return 'other'
+
+
+@login_and_policies_required
+def variant_metadata(request, project_guid):
+    projects, _ = _get_metadata_projects(request, project_guid)
```
Comment (hanars) [inline seqr/views/apis/summary_data_api.py:427]: same as above, copy-pasted from report_api, this is the only line that changed (uses a different helper function to get the allowed projects)

File: seqr/views/apis/summary_data_api_tests.py [hunk 3]
```diff
@@ -701,12 +731,253 @@ def test_sample_metadata_export(self, mock_google_authenticated):
         response = self.client.get(f'{gregor_projects_url}?includeAirtable=true')
         self._has_expected_metadata_response(response, multi_project_individuals, has_airtable=True, has_duplicate=True)
 
+    def test_family_metadata(self):
+        url = reverse(family_metadata, args=['R0003_test'])
+        self.check_collaborator_login(url)
```
Comment (hanars) [inline seqr/views/apis/summary_data_api_tests.py:736]: copy pasted from report_api_tests, only change is changing this from `check_analyst_login` and then adding analyst login back in later

File: seqr/views/apis/summary_data_api_tests.py [hunk 4]
```diff
@@ -701,12 +731,253 @@ def test_sample_metadata_export(self, mock_google_authenticated):
...
+
+        # Test empty project
+        empty_project_url = reverse(family_metadata, args=['R0002_empty'])
+        response = self.client.get(empty_project_url)
+        self.assertEqual(response.status_code, 200)
+        self.assertDictEqual(response.json(), {'rows': []})
+
+    def test_variant_metadata(self):
+        url = reverse(variant_metadata, args=[PROJECT_GUID])
+        self.check_collaborator_login(url)
```
Comment (hanars) [inline seqr/views/apis/summary_data_api_tests.py:819]: copy pasted from report_api_tests, only change is changing this from `check_analyst_login` and then adding analyst login back in later in the test

File: ui/pages/Report/Report.jsx [hunk 5]
```diff
@@ -8,16 +8,12 @@ import { Error404, Error401 } from 'shared/components/page/Errors'
...
 import CustomSearch from './components/CustomSearch'
-import FamilyMetadata from './components/FamilyMetadata'
 import Gregor from './components/Gregor'
 import SeqrStats from './components/SeqrStats'
-import VariantMetadata from './components/VariantMetadata'
 
 export const REPORT_PAGES = [
   { path: 'anvil', component: Anvil },
   { path: 'custom_search', params: '/:searchHash?', component: CustomSearch },
-  { path: 'family_metadata', params: '/:projectGuid?', component: FamilyMetadata },
```
Comment (jklugherz) [inline ui/pages/Report/Report.jsx:19]: Do you think users of the `Reports` tab will care that these are moved?

**Review by jklugherz — APPROVED** [review]

**Pull request merged**